app-arch/bzip2/bzip2-1.0.6.0.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic ltprune multilib-minimal
+
+DESCRIPTION="A high-quality data compressor used extensively by Gentoo Linux"
+HOMEPAGE="http://www.bzip.org/"
+SRC_URI="http://ftp.suse.com/pub/people/sbrabec/bzip2/tarballs/${P}.tar.gz"
+
+LICENSE="BZIP2"
+SLOT="0/1" # subslot = SONAME
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~sparc-fbsd ~x86-fbsd"
+IUSE="static static-libs"
+
+DOCS=( manual.pdf NEWS README.autotools README README.XML.STUFF )
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.0.6-progress.patch
+	"${FILESDIR}"/${PN}-1.0.4-POSIX-shell.patch #193365
+	"${FILESDIR}"/${PN}-1.0.6-mingw.patch #393573
+	"${FILESDIR}"/${PN}-1.0.6-CVE-2016-3189.patch #620466
```

Comment:
Please explain why you have dropped multiple patches.

Reply:
bzip2-1.0.4-makefile-CFLAGS.patch:
* Irrelevant, now that the actual final Makefile is generated from ./configure
bzip2-1.0.6-saneso.patch:
* Irrelevant, you can pass LDFLAGS in when running ./configure
bzip2-1.0.4-man-links.patch:
* Changes this patch would make would be overwritten after ./configure
bzip2-1.0.3-no-test.patch:
* Irrelevant, test is not part of the all job.
bzip2-1.0.6-out-of-tree-build.patch:
* Irrelevant, out-o-tree builds work now with autotools.